_episodes/13-scheduler.md

```diff
@@ -135,12 +131,13 @@ Let's illustrate this by example. By default, a job's name is the name of the sc
 Submit the following job (`{{ site.sched_submit }} {{ site.sched_submit_options }} example-job.sh`):
 
 ```
-#!/bin/bash
+#!/bin/bash -l
 {{ site.sched_comment }} {{ site.sched_flag_name }} new_name
 
 echo 'This script is running on:'
 hostname
 sleep 120
+> echo 'This script has finished successfully'
```

Comment:
Changes on md files should be on its separate branch to propose them upstream.

Reply:
Moved changes to md file into https://github.com/UCL-RITS/hpc-intro/tree/tk/lesson13-upstream-changes and reverted all changes in this branch